pandas/tests/io/test_clipboard.py

```diff
@@ -8,13 +8,7 @@
 from pandas import DataFrame, get_option, read_clipboard
 import pandas.util.testing as tm
 
-from pandas.io.clipboard import PyperclipException, clipboard_get, clipboard_set
```

Comment:
I am not sure we can remove this; for a user this would make these tests fail if the deps are not installed.

Reply:
Good point.

Sorry the title was misleading, this started as a fix, but I'm more debugging now. For now I'm just trying to understand what's being executed, and what are the actual errors. I'm not sure we're only skipping the tests when we think we are. After I remove all the magic and we understand better the problem, we can discuss the fix. I'm not really sure what makes sense at this point.

Reply:
sure happy to have you do that :->

Reply:
let's just disable these directly disable these tests to clear the queue while you work on a patch. (separate PR is ok)